admin/data/sql/migrations/20190621143742.do.create-pupil-results-view.sql

```diff
@@ -0,0 +1,59 @@
+CREATE VIEW [mtc_admin].[vewPupilsResults]
```

Comment:
Should be create or alter view to be idempotent.

Reply:
@jon-shipley I made the change but I think that should be strictly CREATE